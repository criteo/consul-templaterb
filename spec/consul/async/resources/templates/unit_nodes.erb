Nodes
<%
  nodes().each do |n|
%><%= n['Node'] %>
<%
  end
%>
Node: <%= node('consul02.prod')['Node'] %>
ID: <%= node('7a985997-3925-4f18-8613-637c81bd750b')['ID'] %>
