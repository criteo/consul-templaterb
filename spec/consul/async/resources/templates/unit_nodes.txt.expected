Nodes
consul01.prod
consul02.prod
consul03.prod
consul04.prod

Node: consul02.prod
ID: 7a985997-3925-4f18-8613-637c81bd750b
